tests/stdlib/tnetconnect.nim

```diff
@@ -0,0 +1,15 @@
+import std/net
+
+# Issue 15215 - https://github.com/nim-lang/Nim/issues/15215
+proc test() =
+  var
+    ctx = newContext()
+    socket = newSocket()
+
+  wrapSocket(ctx, socket)
+
+  connect(socket, "www.nim-lang.org", Port(443), 5000)
+
```

Comment:
needs to send something like the coded posted on Nim forum

```nim
import net

var ctx = newContext()
var next_socket = newSocket()
wrapSocket(ctx, next_socket)

next_socket.connect("www.google.com", Port(443), 1000)

next_socket.send("GET /")
```

https://forum.nim-lang.org/t/6686